app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt

```diff
@@ -61,26 +61,41 @@ fun HomeDrawer(
         )
 
         topItems.forEach { item ->
-            DrawerItem(data = item.getDrawerData(),
+            DrawerItem(
+                data = item.getDrawerData(),
                 selected = currentRoute == item.route,
                 onItemClick = {
                     navigateToItemInHome(homeNavController, item)
                     scope.launch { drawerState.close() }
-                })
+                }
+            )
         }
 
         Spacer(modifier = Modifier.weight(1f))
 
-        bottomItems.forEach { item ->
-            DrawerItem(data = item.getDrawerData(),
-                selected = currentRoute == item.route,
-                onItemClick = {
-                    scope.launch {
-                        viewModel.navigateTo(item)
-                        drawerState.close()
-                    }
-                })
-        }
+        val settingsItem = NavigationItem.Settings
+        DrawerItem(
+            data = settingsItem.getDrawerData(),
+            selected = currentRoute == settingsItem.route,
+            onItemClick = {
+                scope.launch {
+                    viewModel.navigateTo(settingsItem)
+                    drawerState.close()
+                }
+            }
+        )
+
+        val supportItem = NavigationItem.Support
+        DrawerItem(
+            data = supportItem.getDrawerData(),
+            selected = currentRoute == supportItem.route,
+            onItemClick = {
+                scope.launch {
+                    CustomTabsHelper.launchUrl(homeNavController.context, supportItem.route)
+                    drawerState.close()
+                }
+            }
+        )
 
```

Comment:
to me it looks abit like a code duplication, maybe it would be a good idea to extract this logic to some seperate method that would just generate those DrawerItems, do not know if it is somehow easily achievable though

Reply:
Agree! I wanted to avoid some logic needed. 
I updated the code with an improvement  addressing this.